src/server/JasmineGraphInstanceService.cpp

```diff
@@ -3470,6 +3523,143 @@ int collectBloomFilters(instanceservicesessionargs *sessionargs,
     return 0;
 }
 
+int distributeSharedClustersToWorkers(std::string host, int port, int dataPort,
+                                      std::string fileName, std::string filePath
+                                      ) {
+    Utils utils;
+    Logger instance_logger;
+    bool result = true;
+    std::cout << pthread_self() << " host : " << host << " port : " << port << " DPort : " << dataPort << std::endl;
```

Comment:
Shall we remove this `std::cout` statement? If its required for debugging purpose shall we make it a log statement?